src/handlers/InvitesBtn.ts

```
@@ -1,21 +1,25 @@
 import { ActionRowBuilder, ButtonBuilder } from "@discordjs/builders"
-import { ButtonHandler, ButtonRoute } from "@seedcord/gateway"
+import { ButtonHandler, ButtonRoute, Gated } from "@seedcord/gateway"
```

ESLint

[error] 'Gated' is defined but never used. Allowed unused vars must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] Unable to resolve path to module '`@seedcord/gateway`'.

(import-x/no-unresolved)